metpy/plots/_util.py

```diff
@@ -201,3 +201,82 @@ def colored_line(x, y, c, **kwargs):
     lc = LineCollection(segments, **kwargs)
     lc.set_array(c)
     return lc
+
+
+def gempak_color(c, style='psc'):
```

Comment:
Hate to be the one that gripes about spelling here, but I don't like `gempak_color` for the function name. Function names generally contain a verb (though not necessary). When they don't, they generally name the returned thing (think `degrees` or `radians` from numpy). Therefore, I propose as this does not return a GEMPAK color that it be named `convert_gempak_color`.

Reply:
Seems reasonable. I'll make the tweak.